src/rt/pal/cpu.h

```diff
@@ -29,6 +29,7 @@
 #endif
 
 #include <algorithm>
+#include <functional>
```

Comment:
This was being pulled in from `<thread>` earlier. `std::function` is used in this file, so it is best to make sure the header is also included here.